src/Assets/ListingContainerData.js

```diff
@@ -0,0 +1,140 @@
+export const ListingContainerData = [
```

Comment:
are those only used for testing purpose?

Reply:
yes, it's only for the frontend layout. We will delete them when we deploy on AWS and connect with API Gateway.